openverse_api/catalog/api/models/image.py

```diff
@@ -29,7 +29,11 @@ class Meta:
 
 
 class Image(ImageFileMixin, AbstractMedia):
-    pass
+    """
+    Inherited fields
+    ================
+    category: eg. photograph, digitised artwork & illustration
```

Comment:
Should we include all inherited fields here?

Reply:
We should, but so far only the `category` field has a documentation-worthy note.